lib/SIL/IR/SILFunctionType.cpp

```diff
@@ -1917,9 +1917,11 @@ lowerCaptureContextParameters(TypeConverter &TC, SILDeclRef function,
   // signature from the AST for that.
   auto origGenericSig = function.getAnyFunctionRef()->getGenericSignature();
   auto loweredCaptures = TC.getLoweredLocalCaptures(function);
+  auto *isolatedParam = loweredCaptures.getIsolatedParamCapture();
 
   for (auto capture : loweredCaptures.getCaptures()) {
     if (capture.isDynamicSelfMetadata()) {
+      assert(!isolatedParam);
```

Comment:
What's this assertion for?  Is there a reason we can't have a dynamic self metadata capture *and* an isolated parameter or opaque value?  You don't seem to be using it anyway.

Reply:
I just was being paranoid. I can remove it.